### ISSUE TITLE
PIL Image.py DecompressionBombWarning

### ISSUE THREAD

**Issue by suang2016** [got-10k/siamfc#3]
/home/ubuntu/anaconda3/lib/python3.6/site-packages/PIL/Image.py:2514: DecompressionBombWarning: Image size (99302460 pixels) exceeds limit of 89478485 pixels, could be decompression bomb DOS attack.
DecompressionBombWarning)

Does anybody have this problem?

**Comment by sansanfree** [comment]
yes, i have met with this problems, and ignored it, but my test result with my own trained model is worse than the results provided by authors. so i am confused
